### DISCUSSION TITLE
Add a command to run all existing notebooks

### DISCUSSION THREAD

**Discussion by jinningwang** [CURENT/andes#198] [Ideas]
Hi Hantao,

I noticed there is a package that can run ``ipynb`` file in a py code: [runipy](https://github.com/paulgb/runipy). Maybe we can add a preparation check like ``andes prep demo`` to run all notebook files.

Regards,
Jinning

**Comment by cuihantao** [comment]
If it’s for checking if notebooks work, the CI on GitHub already does it. See the command in the GitHub workflow configuration file if you want to run it locally, after installing a few packages.

Regards,
Hantao Cui

**Reply by jinningwang** [reply]
Got it, thanks!
